docs/docsite/rst/inventory_guide/intro_dynamic_inventory.rst

```diff
@@ -120,7 +120,7 @@ Inventory script example: OpenStack
 
 If you use an OpenStack-based cloud, instead of manually maintaining your own inventory file, you can use the ``openstack_inventory.py`` dynamic inventory to pull information about your compute instances directly from OpenStack.
 
-You can download the latest version of the OpenStack inventory script `here <https://raw.githubusercontent.com/openstack/ansible-collections-openstack/master/scripts/inventory/openstack_inventory.py>`_.
+You can download the latest version of the OpenStack inventory script `here <https://raw.githubusercontent.com/openstack/ansible-collections-openstack/master/plugins/inventory/openstack.py>`_.
```

Comment:
This is very likely **NOT** an inventory script, but an inventory plugin. Inventory plugins must be used differently from inventory scripts, so this change is most likely wrong.

I'm wondering whether it isn't better to remove the OpenStack inventory script examples completely.